cli/test/lib/errors_spec.js

```diff
@@ -16,14 +16,18 @@ describe('errors', function () {
   describe('individual', () => {
     it('has the following errors', () => {
       return snapshot(Object.keys(errors))
-    }
-    )
+    })
   })
 
   context('.errors.formErrorText', function () {
     it('returns fully formed text message', () => {
-      return snapshot(formErrorText(missingXvfb))
-    }
-    )
+      expect(missingXvfb).to.be.an('object')
+
+      return formErrorText(missingXvfb)
```

Comment:
had to rewrite this, was hanging, hmm, snap-shot-it no longer allows a promise?!